docker-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
   dbt:
     platform: linux/amd64
     image: medicmobile/dataemon:latest
+    build: ./dbt/
```

Comment:
We already have `image` defined do we need `build` as well?

Reply:
I think we need build because we want docker to build the image from that directory. Othwerwise, changes to the dockerfile and dbt-run.py won't get picked up and it willl always use the image built from the dataemon repository. The github build action then pushes the changes to the dockerhub repository. Its similar to medicmobile/cht-logstash-couchdb just above

Reply:
I was questioning the need for both in the same container for the same environment i.e. would it make sense to have a `docker-compose-prod` file that uses the image and not the build? I guess if we are using kubernetes for prod then it shouldn't be an issue right?

Reply:
When I checked the [docs](https://docs.docker.com/compose/compose-file/build/#using-build-and-image), I missed the part where the image is built only if the image is missing in the registry. Thanks for the clarification.